tests/shardddl1/lib.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -eu
```

Comment:
Can we reuse **one** `lib.sh` for all these three cases?

Reply:
address it in e4fd8d5